lib/components/Form/AutoComplete/index.jsx

```diff
@@ -5,7 +5,7 @@ const AutoComplete = ({ items, onClick, searchValue }) => (
   <ul className="autocomplete">
     {
       items.map((item) => {
-        const formattedText = { __html: item.text.replace(searchValue, `<b>${searchValue}</b>`) };
+        const formattedText = { __html: item.text.replace(new RegExp(searchValue, 'ig'), `<b>${searchValue}</b>`) };
```

Comment:
Elaborate

Reply:
In regex i = case insensitive mode and g = global search I.E replace all instances